istioctl/cmd/analyze.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"istio.io/istio/galley/pkg/config/analysis/local"
+
+	"github.com/spf13/cobra"
+)
+
+// Analyze command
+func Analyze() *cobra.Command {
+	return &cobra.Command{
+		Use:     "analyze",
+		Short:   "Analyze Istio configuration",
+		Example: `istioctl experimental analyze <directory>`,
```

Comment:
This is currently misleading: if you actually just give it a directory, it will fail. 
```
crwilson@crwilson:~/go/src/istio.io/istio$ /usr/local/google/home/crwilson/go/out/linux_amd64/release/istioctl experimental analyze ~/tmp2/test_config
Error: read /usr/local/google/home/crwilson/tmp2/test_config: is a directory

crwilson@crwilson:~/go/src/istio.io/istio$ /usr/local/google/home/crwilson/go/out/linux_amd64/release/istioctl experimental analyze ~/tmp2/test_config/*
[EIST0020](VirtualService/httpbin) Referenced Gateway not found: "httpbin-gateway2"
```